shared/src/proxies/getConsolidatedCasesByCaseProxy.js

```diff
@@ -10,10 +10,10 @@ const { get } = require('./requests');
  */
 exports.getConsolidatedCasesByCaseInteractor = (
   applicationContext,
-  { docketNumber },
+  { docketNumber, pickFields },
 ) => {
   return get({
     applicationContext,
-    endpoint: `/cases/${docketNumber}/consolidated-cases`,
+    endpoint: `/cases/${docketNumber}/consolidated-cases?fields=${pickFields}`,
```

Comment:
I think the approach we normally take is the pass a params object instead of setting the query string manually, like so:

```
return get({
    applicationContext,
    endpoint: '/case-documents/opinion-search',
    params: {
      ...omit(searchParams, 'opinionTypes'),
      opinionTypes: opinionTypesQuery,
    },
  });
  ```